integrations/nlu-rasa/src/RasaNlu.ts

```diff
@@ -3,19 +3,30 @@ import {
   AxiosRequestConfig,
   AxiosResponse,
   DeepPartial,
-  Extensible,
+  EntityMap,
   HandleRequest,
   Jovo,
-  JovoError,
   NluData,
   NluPlugin,
   PluginConfig,
 } from '@jovotech/framework';
-import { RasaResponse } from './interfaces';
+import { RasaEntity, RasaIntent, RasaResponse } from './interfaces';
 
 export interface RasaNluConfig extends PluginConfig {
```

Comment:
In v4, the Config-type of a plugin is supported to exactly reflect the config as it should be, meaning if the integration needs a `serverUrl` like here for example, then it should not be optional in the `RasaNluConfig`.
Only properties that are really optional and not required for the integration to work are now supposed to be optional.

Reply:
all config elements are now mandatory. alternative intent mapping should have been optional, depending on if the config option is given or not. but i am totally fine with having it as a fixed part. The change led to a considerable code restructure.